packages/backend-app-api/src/wiring/BackendInitializer.ts

```diff
@@ -437,7 +457,9 @@ export class BackendInitializer {
 
   async #getPluginLifecycleImpl(
     pluginId: string,
-  ): Promise<LifecycleService & { startup(): Promise<void> }> {
+  ): Promise<
+    LifecycleService & { startup(): Promise<void>; shutdown(): Promise<void> }
```

Comment:
see the `typeof service.startup === 'function'` check below - you will want to add the shutdown check there too